README.md

```diff
@@ -681,12 +681,13 @@ column `mytimestamp`:
         "type" : "jdbc",
         "jdbc" : {
 	        "url" : "jdbc:mysql://localhost:3306/test",
+            "statefile" : "statefile.json",
 	        "user" : "",
 	        "password" : "",
             "sql" : [
                 {
-                    "statement" : "select * from \"products\" where \"mytimestamp\" > ?",
-                    "parameter" : [ "$metrics.lastexecutionstart" ]
+                    "statement" : "select * from products where mytimestamp > ?",
+                    "parameter" : [ "$metrics.lastexecutionend" ]
```

Comment:
Should be lastexecutionstart because in this case we will missed inserted data between 06:37:09.165 and 06:37:09.501